manim/cli/init/commands.py

```diff
@@ -0,0 +1,31 @@
+"""Manim's init subcommand.
+
+Manim's init subcommand is accessed in the command-line interface via ``manim
+init``. Here you can specify options, subcommands, and subgroups for the init
+group.
+
+"""
+from pathlib import Path
+
+import click
+
+from ...constants import CONTEXT_SETTINGS, EPILOG
+from ...utils.file_ops import copy_template_files
+
+
+@click.command(
+    context_settings=CONTEXT_SETTINGS,
+    epilog=EPILOG,
+)
+def init():
+    """Sets up a project in current working directory.
```

Comment:
Since this also appears to be the help string for this command, maybe it'd be better to specify that the project is created with default settings?
```suggestion
    """Sets up a project in current working directory with default settings.
```

another option might be to explicitly add a `help` argument to the decorator (like with `new`)

Reply:
I used the help argument with the decorator but, jsonvilanueva said that click uses docstrings as help.

Reply:
if i type manim init -h it shows the docstring

Reply:
sorry, my English is bad.